README.md

```diff
@@ -94,6 +94,7 @@ Iterable's API, see the [API Overview](https://support.iterable.com/hc/articles/
 
 | Method Name                                                                       | Description |
 | --------------------------------------------------------------------------------- | ----------- |
+| [`baseIterableRequest`](#baseiterablerequest)                                     |  |
```

Comment:
Could use a lil help with the copy. Wasn't sure what to add as the description here.

Reply:
In the past, Docs (Brad) had helped with writing descriptions for this kind of description. However, he was also a bit more involved and wanted a hand in some of the more technical documentation. Up to you whether you think we should just write up something ourselves or get Docs help on it, since the README is kind of an in-between space between Eng and Docs haha